<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>knn</class>
 <widget class="QMainWindow" name="knn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>visualizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>CIFAR10 Test Set</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="6">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Knn - properties</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2" colspan="2">
      <widget class="QSpinBox" name="numberBox">
       <property name="maximum">
        <number>60000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPicture">
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>61</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Label:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>knn:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Acc:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="accResult">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QRadioButton" name="L1">
       <property name="text">
        <string>L1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QRadioButton" name="L2">
       <property name="text">
        <string>L2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>k:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QSpinBox" name="kBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <widget class="QPushButton" name="evalButton">
       <property name="text">
        <string>Evaluate acc.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="2">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
